iTaylorCodes/Capstone-2-Neighborhood-fe
# Add WalkScore API Integration

### What does this PR do?
- This PR integrates the WalkScore API to show scores for each neighborhood.
- This PR closes #19 
### Description of Task to be completed
- Add `Neighborhood` component, `Scores` component, and `Score` component. Setup WalkScoreAPI class to handles calls to Walkscore's API.
### How should this be manually tested?
- Visit `/` and search for a city. Verify it navigates to the `/neighborhood/:neighborhood` route and displays scores for the city.

https://user-images.githubusercontent.com/76837421/148005652-c1924490-d283-4424-97d3-fdcca69bc561.mov
